test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -90,3 +97,8 @@ spec:
           hostPath:
             path: /sys
             type: Directory
+      # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+      # See "special case". This will tolerate everything. Node component should
+      # be scheduled on all nodes.
+      tolerations:
+      - operator: Exists
```

Comment:
So the daemonset pod will be scheduled to any node including the one that is marked as "NoSchedulable"? For example, e2e cluster windows node might be tainted, with this toleration,  the pod will be scheduled to windows node too. Just want to understand what is the desired behavior here.

Reply:
yes, this is to make the daemonset schedule on all nodes even when a user taints their node with arbitrary things (for other user workloads). Yes, for windows this would be scheduled too if the only guard is a taint - is this going to break any assumptions?

Reply:
don't we need to add a nodeselector for linux nodes? We're going to need a different DaemonSet to run on Windows.

Reply:
so basically this pod will fail to run Windows node. I am not sure whether we have some assumption or verification about pods being running successfully. 
I think nodeselector can be used since we have different labels for linux and windows nodes.

Reply:
I'm waiting on confirmation that a linux label will always exist on Kubernetes distributions before adding a node selector to the deployment YAMLs

Reply:
added the label selector like in the original repo https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/475

Reply:
I think the label is added by kubelet which should be very reliable unless user explicitly deletes it.